crmsh/ra.py

```diff
@@ -50,8 +50,9 @@ def ra_classes():
         return cache.retrieve("ra_classes")
     if can_use_crm_resource():
         l = crm_resource("--list-standards")
+        l = [x for x in l if x not in ("lsb", "service")]
     else:
-        l = ["heartbeat", "lsb", "ocf", "stonith", "systemd"]
```

Comment:
How is `heartbeat` related to `lsb`?

Reply:
"heartbeat" is used by the current ocf1.0 and 1.1 standard and located at
/usr/lib/ocf/resource.d/heartbeat

Reply:
```
# crm_resource --list-standards
ocf
lsb
service
systemd
stonith
```
Just synced with the output of --list-standards